### PR TITLE
Cleaning tools and "setup and run"

### PR DESCRIPTION
**Pull request by Jgmedina95**

### CODE REVIEW

**Review by SamCox822 — COMMENTED** [review]
In mdagent/tools/search_tools.py
can you change the prompt summary to use the BaseLanguageModel from langchain (it's also used in tools.py). We need to keep llm model types the same throughout

File: mdagent/tools/search_tools.py
Comment (SamCox822) [inline mdagent/tools/search_tools.py:1]: Can you change the prompt summary to use the BaseLanguageModel from langchain (it's also used in tools.py). We need to keep llm model types the same throughout

File: notebooks/set_and_run_example.ipynb
Comment (SamCox822) [inline notebooks/set_and_run_example.ipynb:1]: Can you make this a full example that makes the params file and then runs?

File: environment.yaml
Comment (SamCox822) [inline environment.yaml:1]: Since you have to add openmm and pdbfixer via conda, can you add those instructions on the readme under Installation?

**Review by SamCox822 — CHANGES_REQUESTED** [review]

File: mdagent/tools/search_tools.py
Comment (Jgmedina95) [reply mdagent/tools/search_tools.py:1]: done

File: notebooks/set_and_run_example.ipynb
Comment (Jgmedina95) [reply notebooks/set_and_run_example.ipynb:1]: done

File: environment.yaml
Comment (Jgmedina95) [reply environment.yaml:1]: done

**Review by SamCox822 — COMMENTED** [review]
I did a little rearranging and added a notebook that reflects a full user story.

**Review by SamCox822 — APPROVED** [review]

**Pull request merged**